src/core_modules/capture-core/components/Pages/ViewEvent/ViewEventComponent/ViewEvent.container.js

```diff
@@ -43,6 +43,9 @@ const mapDispatchToProps = (dispatch: ReduxDispatch) => ({
     onBackToAllEvents: () => {
         dispatch(startGoBackToMainPage());
     },
+    onBackToViewEvent: () => {
+        dispatch(cancelEditEventDataEntry());
```

Comment:
@henrikmv Should we also dispatch `setCurrentDataEntry` just like is done when clicking the form's [Cancel](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/EditEventDataEntry.container.js#L128-L131) button?